extension/js/background.js

```diff
@@ -41,10 +39,28 @@ async function makeRelayAddressForTargetElement(info, tab) {
   });
 }
 
+browser.menus.create({
+  id: "fx-private-relay-generate-alias",
+  title: "Generate Email Alias",
+  contexts: ["editable"]
+});
+
 browser.menus.onClicked.addListener( async (info, tab) => {
   switch (info.menuItemId) {
     case "fx-private-relay-generate-alias":
       await makeRelayAddressForTargetElement(info, tab);
       break;
   }
 });
+
+browser.runtime.onMessage.addListener(async (m) => {
+  let response;
+
+  switch (m.method) {
+    case "makeRelayAddress":
+      response = await makeRelayAddress();
+      break;
+  }
+
+  return response;
```

Comment:
Let's assueme apiToken is not null, and `makeRelayAddress` returns a `Response` object.
`Response` is not serializable and it cannot be sent from the background.js to the popup.js.
What you want is probably:

`return await response.text()`;

Reply:
This was it; thanks!!